lib/backpex/html/core_components.ex

```diff
@@ -13,4 +13,35 @@ defmodule Backpex.HTML.CoreComponents do
     <span class={[@name, @class]} {@rest} />
     """
   end
+
+  @doc """
+  Renders a filter_badge component.
+  """
+  attr :clear_event, :string, doc: "event name for removing the badge"
```

Comment:
We should add a default here:

```suggestion
  attr :clear_event, :string, default: "clear-filter", doc: "event name for removing the badge"
```

Reply:
Good idea 👍